releases/release-1.31/release-notes/maps/pr-124464-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 124464
+releasenote:
+  text: 'kubeadm: use output/v1alpha3 to print structural output for the commands
```

```suggestion
  text: 'kubeadm: Used output/v1alpha3 to print structural output for the commands
```
https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653311879